### PR TITLE
fix: '-e/-f' CLI parameters are not handling session variables correctly

### PR DESCRIPTION
**Pull request by spena**
### Description 

There are three fixes in this PR:
* fix: allow '-e' parameter to use variable substitution
* fix: SHOW VARIABLES is not working with RUN SCRIPT and '-f' parameter
* fix: RUN SCRIPT and '-f' parameter do not handle PRINT/SELECT queries

### Testing done 
Added unit tests and verified manually

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksqldb-cli/src/main/java/io/confluent/ksql/cli/Cli.java [hunk 1]
```diff
@@ -354,22 +352,21 @@ private boolean isVariableSubstitutionEnabled() {
...
 
-  private List<ParsedStatement> substituteVariables(final List<ParsedStatement> statements) {
+  private ParsedStatement substituteVariables(final ParsedStatement statement) {
     if (isVariableSubstitutionEnabled()) {
-      return statements.stream()
-          .map(stmt -> VariableSubstitutor.substitute(stmt, sessionVariables))
-          .flatMap(replacedSql -> KSQL_PARSER.parse(replacedSql).stream())
-          .collect(Collectors.toList());
+      final String replacedStmt = VariableSubstitutor.substitute(statement, sessionVariables);
+      return KSQL_PARSER.parse(replacedStmt).get(0);
```
Comment (agavra) [inline ksqldb-cli/src/main/java/io/confluent/ksql/cli/Cli.java:358]: should we use `Iterables.getOnlyElement` here to make sure it fails if there's more than 1?

File: ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/VariablesList.java [hunk 2]
```diff
@@ -80,7 +80,7 @@ public String toString() {
   @JsonCreator
   public VariablesList(
       @JsonProperty("statementText") final String statementText,
-      @JsonProperty("properties") final List<Variable> variables
+      @JsonProperty("variables") final List<Variable> variables
```
Comment (agavra) [inline ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/VariablesList.java:83]: this has never been released, right? should be safe to change

**Review by agavra — APPROVED** [review]
LGTM

File: ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/VariablesList.java (hunk 2 above)
Comment (spena) [reply ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/VariablesList.java:83]: Yeap, never released, so we're good here.

**Pull request merged**